hooks/post_gen_project.sh

```diff
@@ -43,8 +43,8 @@ fi
 
 if echo "{{ cookiecutter.add_graphql }}" | grep -iq "^n"; then
     rm -rf {{ cookiecutter.main_module }}/graphql
-    rm -rf {{ cookiecutter.main_module }}/tests/integration/graphql
     rm -rm {{ cookiecutter.main_module }}/docs/graphql
+    rm -rf tests/graphql
```

Comment:
```suggestion
    rm -rf {{ cookiecutter.main_module }}/tests/graphql

Reply:
tests are outside main_module

 ![image](https://user-images.githubusercontent.com/3005382/206157483-c44678ec-4c06-426d-9dbb-9f352a6585d0.png)

Reply:
Cool. Feel free to merge this in `add-graphene` branch then.